.github/workflows/codeql.yml

```
@@ -0,0 +1,42 @@
+name: CodeQL
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)